src/Workspaces/Core/Portable/Serialization/PooledList.cs

```diff
@@ -14,31 +14,6 @@ namespace Microsoft.CodeAnalysis.Serialization;
 /// </summary>
 internal static class Creator
```

Comment:
nit: debatable whether this class even needs to exist if it's a single method with a single line of code

Reply:
it was called in a bunchh of locations.  i ididn't want to updat those all now.  leaving as is.

Reply:
No worries, I thought I only saw a couple, but I could be wrong